src/services/i18n.ts

```diff
@@ -71,7 +90,9 @@ export const resources = {
             password: "Passwort",
             repeatPassword: "Passwort Wiederholen",
             username: "Benutzername",
+            description: "Beschreibung",
             displayName: "Anzeige Name",
+            tripName: "Trip Name",
```

Comment:
I'm currently thinking if 'Trip' is the right translation 🤔 for german. Just a suggestion
```suggestion
            tripName: "Reisename",
```
Should we change we will also have to change it in other translations.